eternalcore-core/src/main/java/com/eternalcode/core/translation/Translation.java

```diff
@@ -133,6 +135,11 @@ interface ChatSection {
         Notice tellrawNoSaved();
         Notice tellrawMultipleSent();
         Notice tellrawCleared();
+        Notice alertQueueAdded();
+        Notice alertQueueRemoved();
+        Notice alertQueueCleared();
+        Notice alertQueueEmpty();
+        Notice alertQueueSent();
```

Comment:
I must admit that your names are much better and more meaningful than tellraw (queue) :P